_layouts/documentation.html

```diff
@@ -43,7 +43,13 @@
                     <a href="/documentation/commandline">Command-line</a></li>
                     <li
                     {% if page.menu == 'api' %} class="active" {% endif %}>
-                    <a href="/documentation/api">API (Java / Android)</a></li>
+                    <a href="/documentation/api">API</a></li>
+                    <li
+                    {% if page.menu == 'api' %} class="active" {% endif %}>
+                    <a href="/documentation/api#android">API (Android)</a></li>
+                    <li
+                    {% if page.menu == 'api' %} class="active" {% endif %}>
+                    <a href="/documentation/api#spring">API (Spring)</a></li>
                     <li
```

Comment:
I was conflicted about this. I found myself searching at random for the android info many times, however. It would be nice if it was more directly accessible. 

For me, the Java API is the supremely important part of the doc. The other pages, I have barely read.